### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Bento007**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,3 +18,10 @@ expected to abide by the following guidelines.
 
 - Keep the build simple: Automated CI checks that are fragile or don't
   serve a clear agreed upon purpose will be removed.
+
+- All Pull Request comments must be addressed, even after merge.
+
+- All code must be review by at least 1 other team member.
```
Comment (kislyuk) [inline CONTRIBUTING.md]: s/review/reviewed/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,3 +18,10 @@ expected to abide by the following guidelines.
...
+
+- All Pull Request relating to an issue must include "connected to
```
Comment (kislyuk) [inline CONTRIBUTING.md]: s/Request/Requests/

File: CONTRIBUTING.md (hunk 1 above)
Comment (Bento007) [reply CONTRIBUTING.md]: corrected

File: CONTRIBUTING.md (hunk 2 above)
Comment (Bento007) [reply CONTRIBUTING.md]: corrected

**Pull request merged**